pal_tiago/tiago.xml

```diff
@@ -0,0 +1,222 @@
+<mujoco model="tiago">
+
+  <compiler angle="radian" meshdir="./assets/" autolimits="true"/>
+
+  <option timestep="0.001" integrator="implicit" />
+
+  <asset>
+    <mesh name="sick_tim551" file="sensors/sick_tim551.stl"/>
+    <mesh name="srf05" file="sensors/srf05.stl"/>
+    <mesh name="base" file="base/base.stl"/>
+    <mesh name="base_ring" file="base/base_ring.stl"/>
+    <mesh name="antenna" file="objects/antenna.stl"/>
+    <mesh name="wheel" file="wheels/wheel.stl"/>
+    <mesh name="caster_1" file="wheels/caster_1.stl"/>
+    <mesh name="caster_2" file="wheels/caster_2.stl"/>
+    <mesh name="torso_fix" file="torso/torso_fix.stl"/>
+    <mesh name="torso_lift_with_arm" file="torso/torso_lift_with_arm.stl"/>
+    <mesh name="head_1" file="head/head_1.stl"/>
+    <mesh name="head_2" file="head/head_2.stl"/>
+    <mesh name="arm_1" file="arm/arm_1.stl"/>
+    <mesh name="arm_2" file="arm/arm_2.stl"/>
+    <mesh name="arm_3" file="arm/arm_3.stl"/>
+    <mesh name="arm_4" file="arm/arm_4.stl"/>
+    <mesh name="arm_5-wrist-2017" file="arm/arm_5-wrist-2017.stl"/>
+    <mesh name="arm_6-wrist-2017" file="arm/arm_6-wrist-2017.stl"/>
+    <mesh name="gripper_link" file="gripper_link.stl"/>
+    <mesh name="gripper_finger_link" file="gripper_finger_link.stl" scale="-1 -1 1"/>
+    <mesh name="gripper_finger_link1" file="gripper_finger_link.stl"/>
+  </asset>
+
+  <default class="main">
+    <geom contype="0" conaffinity="1" group="1"/>
+    <default class="arm">
+      <geom contype="1" conaffinity="1"/>
+    </default>
+  </default>
+
+  <worldbody>
+    <body name="base_link" pos="0 0 0">
+      <joint name="reference" type="free"/>
+      <geom pos="0 0 0.0985" type="mesh" density="0" rgba="1 1 1 1" mesh="base"/>
```

Comment:
Move all the rgba definitions into materials and define them in `<assets>`.

Reply:
Done